policy/pa.go

```diff
@@ -28,10 +28,11 @@ import (
 type AuthorityImpl struct {
 	log blog.Logger
 
-	blocklist              map[string]bool
-	exactBlocklist         map[string]bool
-	wildcardExactBlocklist map[string]bool
-	blocklistMu            sync.RWMutex
+	domainBlocklist       map[string]bool
+	fqdnBlocklist         map[string]bool
+	wildcardFqdnBlocklist map[string]bool
+	prefixBlocklist       []netip.Prefix
```

Comment:
The renames of the domain / fqdn fields are quite nice! The prefix blocklist could use a correspondingly clear name:


```suggestion
	domainBlocklist       map[string]bool
	fqdnBlocklist         map[string]bool
	wildcardFqdnBlocklist map[string]bool
	ipPrefixBlocklist       []netip.Prefix
```